### PR TITLE
Plugin list: align disabled autoupdate message with other messages

### LINKED ISSUE #17260: Plugins: Misaligned labels on all-sites view of single plugin
#### Steps to reproduce
1. Set up a Jetpack site and disable autoupdates via [setting](https://jetpack.com/support/site-management/#file-update-disabled) the `DISALLOW_FILE_MODS` constant in `wp-config`
2. Install a plugin on that site
3. Visit the plugin detail page (for all sites) for the plugin within Calypso (e.g. `/plugins/:plugin`)
4. Expand the "Installed on" card for that site
5. Observe misaligned label for "Autoupdates disabled"

#### What I expected
All labels to be lined up properly.

#### What happened instead
"Autoupdates disabled" label is missing some margin on the left.
screen_shot_2017-08-16_at_4_56_56_pm

#### Browser / OS version
Chrome, macOS

#### Context / Source
#reallife #dogfooding #manual-testing

### PR DESCRIPTION
**Pull request by jeherve**
Fixes #17260

**Before:**

screenshot 2017-09-15 at 16 27 40

**After:**

screenshot 2017-09-15 at 16 27 30

**To test:**

Follow the instructions in the original bug report:
https://github.com/Automattic/wp-calypso/issues/17260#issue-250784771

### CODE REVIEW

File: client/my-sites/plugins/plugin-action/style.scss [hunk 1]
```diff
@@ -64,7 +65,6 @@
 .plugin-action__disabled-info.info-popover {
 	float: right;
 	height: 16px;
-	width: 24px;
```
Comment (iamtakashi) [inline client/my-sites/plugins/plugin-action/style.scss]: Why does this need to be removed? It seems to cause a misalignment on the manage page when the viewport is less than 1040px.

screen shot 2017-09-20 at 14 26 55

Comment (jeherve) [reply client/my-sites/plugins/plugin-action/style.scss]: Without it the icons aren't aligned on the plugin detail page. :(

I'm not sure what the best alternative is here.

Comment (iamtakashi) [reply client/my-sites/plugins/plugin-action/style.scss]: @jeherve I'd suggest keeping the width setting as it is, and instead, change the `translateX` value to `0` like this `transform: translate(0, -2px);` for `.plugin-action__disabled-info.info-popover .gridicons-info-outline` (Just a few lines below)

Comment (jeherve) [reply client/my-sites/plugins/plugin-action/style.scss]: It seems to be hard to find something that will work everywhere.

With 
```css
.plugin-action__disabled-info.info-popover .gridicons-info-outline {
	transform: translate(2px, -2px);
}
```
- Plugin detail page looks good:
screenshot 2017-09-21 at 15 51 15
- Plugins list is not perfectly aligned:
screenshot 2017-09-21 at 15 51 39

With
```css
.plugin-action__disabled-info.info-popover .gridicons-info-outline {
	transform: translate(0, -2px);
}
```

- Plugin detail page has a small issue:
screenshot 2017-09-21 at 15 47 05
- Plugins list looks good:
screenshot 2017-09-21 at 15 46 18

**Comment by iamtakashi** [comment]
OK, can I take over this? I'll see what I can do.

**Comment by iamtakashi** [comment]
This should fix the issue. @jeherve if this is working for you too, go ahead and merge it.

screen shot 2017-09-25 at 23 53 02

screen shot 2017-09-25 at 23 52 41

**Comment by jeherve** [comment]
Looks good to me, thank you! Merging.

**Comment by jsnajdr** [comment]
This patch fixes one bug, but introduces another. Select all sites and go to an info page of some plugins. At the bottom, there is a list of sites where the plugin is installed. And it now looks like this:

plugin-remove-all-sites

@iamtakashi Any attempt to fix the plugin actions CSS is doomed to fail 🤣

**Comment by iamtakashi** [comment]
Ugh, sorry. Yeah, this part of CSS in Calypso is exactly like this 😠 

giphy-downsized

**Pull request merged**